### PR TITLE
es: Format /web/{!api,css,html,http,javascript} using Prettier

### PR DESCRIPTION
**Pull request by queengooborg**
This PR formats the small folders within the `/web` folder of the Spanish locale using Prettier.

### CODE REVIEW

File: files/es/web/progressive_web_apps/tutorials/js13kgames/app_structure/index.md [hunk 1]
```diff
@@ -243,10 +242,10 @@ self.addEventListener("install", function (e) {
   e.waitUntil(
     caches.open(cacheName).then(function (cache) {
       console.log(
-        "[Servicio Worker] Almacena todo en caché: contenido e intérprete de la aplicación"
+        "[Servicio Worker] Almacena todo en caché: contenido e intérprete de la aplicación",
```
Comment (Graywolf9) [inline files/es/web/progressive_web_apps/tutorials/js13kgames/app_structure/index.md:245]:
```suggestion
        "[Servicio Worker] Almacena todo en caché: contenido e intérprete de la aplicación
```

File: files/es/web/progressive_web_apps/tutorials/js13kgames/app_structure/index.md [hunk 2]
```diff
@@ -243,10 +242,10 @@ self.addEventListener("install", function (e) {
...
       );
       return cache.addAll(contentToCache);
-    })
+    }),
```
Comment (Graywolf9) [inline files/es/web/progressive_web_apps/tutorials/js13kgames/app_structure/index.md:248]:
```suggestion
    })
```

File: files/es/web/progressive_web_apps/tutorials/js13kgames/app_structure/index.md [hunk 3]
```diff
@@ -263,14 +262,14 @@ self.addEventListener("fetch", function (e) {
         fetch(e.request).then(function (response) {
           return caches.open(cacheName).then(function (cache) {
             console.log(
-              "[Servicio Worker] Almacena el nuevo recurso: " + e.request.url
+              "[Servicio Worker] Almacena el nuevo recurso: " + e.request.url,
```
Comment (Graywolf9) [inline files/es/web/progressive_web_apps/tutorials/js13kgames/app_structure/index.md:265]:
```suggestion
              "[Servicio Worker] Almacena el nuevo recurso: " + e.request.url
```

File: files/es/web/progressive_web_apps/tutorials/js13kgames/app_structure/index.md [hunk 4]
```diff
@@ -263,14 +262,14 @@ self.addEventListener("fetch", function (e) {
...
             );
             cache.put(e.request, response.clone());
             return response;
           });
         })
       );
-    })
+    }),
```
Comment (Graywolf9) [inline files/es/web/progressive_web_apps/tutorials/js13kgames/app_structure/index.md:272]:
```suggestion
    })
```

File: files/es/web/progressive_web_apps/tutorials/js13kgames/offline_service_workers/index.md [hunk 5]
```diff
@@ -106,10 +106,10 @@ self.addEventListener("install", (e) => {
   e.waitUntil(
     caches.open(cacheName).then((cache) => {
       console.log(
-        "[Servicio Worker] Almacena todo en caché: contenido e intérprete de la aplicación"
+        "[Servicio Worker] Almacena todo en caché: contenido e intérprete de la aplicación",
```
Comment (Graywolf9) [inline files/es/web/progressive_web_apps/tutorials/js13kgames/offline_service_workers/index.md:109]:
```suggestion
        "[Servicio Worker] Almacena todo en caché: contenido e intérprete de la aplicación"
```

File: files/es/web/progressive_web_apps/tutorials/js13kgames/offline_service_workers/index.md [hunk 6]
```diff
@@ -106,10 +106,10 @@ self.addEventListener("install", (e) => {
...
       );
       return cache.addAll(contentToCache);
-    })
+    }),
```
Comment (Graywolf9) [inline files/es/web/progressive_web_apps/tutorials/js13kgames/offline_service_workers/index.md:112]:
```suggestion
    })
```

File: files/es/web/progressive_web_apps/tutorials/js13kgames/offline_service_workers/index.md [hunk 7]
```diff
@@ -150,14 +150,14 @@ self.addEventListener("fetch", (e) => {
         fetch(e.request).then((response) => {
           return caches.open(cacheName).then((cache) => {
             console.log(
-              "[Servicio Worker] Almacena el nuevo recurso: " + e.request.url
+              "[Servicio Worker] Almacena el nuevo recurso: " + e.request.url,
```
Comment (Graywolf9) [inline files/es/web/progressive_web_apps/tutorials/js13kgames/offline_service_workers/index.md:153]:
```suggestion
              "[Servicio Worker] Almacena el nuevo recurso: " + e.request.url
```

File: files/es/web/progressive_web_apps/tutorials/js13kgames/offline_service_workers/index.md [hunk 8]
```diff
@@ -150,14 +150,14 @@ self.addEventListener("fetch", (e) => {
...
             );
             cache.put(e.request, response.clone());
             return response;
           });
         })
       );
-    })
+    }),
```
Comment (Graywolf9) [inline files/es/web/progressive_web_apps/tutorials/js13kgames/offline_service_workers/index.md:160]:
```suggestion
    })
```

File: files/es/web/progressive_web_apps/tutorials/js13kgames/offline_service_workers/index.md [hunk 9]
```diff
@@ -187,7 +187,7 @@ self.addEventListener("install", (e) => {
   e.waitUntil(
     caches.open("js13kPWA-v2").then((cache) => {
       return cache.addAll(contentToCache);
-    })
+    }),
```
Comment (Graywolf9) [inline files/es/web/progressive_web_apps/tutorials/js13kgames/offline_service_workers/index.md:190]:
```suggestion
    })
```

File: files/es/web/progressive_web_apps/tutorials/js13kgames/offline_service_workers/index.md [hunk 10]
```diff
@@ -207,9 +207,9 @@ self.addEventListener("activate", (e) => {
           if (key !== cacheName) {
             return caches.delete(key);
           }
-        })
+        }),
```
Comment (Graywolf9) [inline files/es/web/progressive_web_apps/tutorials/js13kgames/offline_service_workers/index.md:210]:
```suggestion
        })
```

File: files/es/web/progressive_web_apps/tutorials/js13kgames/offline_service_workers/index.md [hunk 11]
```diff
@@ -207,9 +207,9 @@ self.addEventListener("activate", (e) => {
...
       );
-    })
+    }),
```
Comment (Graywolf9) [inline files/es/web/progressive_web_apps/tutorials/js13kgames/offline_service_workers/index.md:212]:
```suggestion
    })
```

File: files/es/web/progressive_web_apps/tutorials/js13kgames/re-engageable_notifications_push/index.md [hunk 12]
```diff
@@ -203,7 +203,7 @@ const webPush = require("web-push");
 if (!process.env.VAPID_PUBLIC_KEY || !process.env.VAPID_PRIVATE_KEY) {
   console.log(
     "Debes configurar las variables de entorno VAPID_PUBLIC_KEY y " +
-      "VAPID_PRIVATE_KEY. Puedes utilizar las siguientes: "
+      "VAPID_PRIVATE_KEY. Puedes utilizar las siguientes: ",
```
Comment (Graywolf9) [inline files/es/web/progressive_web_apps/tutorials/js13kgames/re-engageable_notifications_push/index.md:206]:
```suggestion
      "VAPID_PRIVATE_KEY. Puedes utilizar las siguientes: "
```

File: files/es/web/progressive_web_apps/tutorials/js13kgames/re-engageable_notifications_push/index.md [hunk 13]
```diff
@@ -212,7 +212,7 @@ if (!process.env.VAPID_PUBLIC_KEY || !process.env.VAPID_PRIVATE_KEY) {
 webPush.setVapidDetails(
   "https://github.com/mdn/serviceworker-cookbook/",
   process.env.VAPID_PUBLIC_KEY,
-  process.env.VAPID_PRIVATE_KEY
+  process.env.VAPID_PRIVATE_KEY,
```
Comment (Graywolf9) [inline files/es/web/progressive_web_apps/tutorials/js13kgames/re-engageable_notifications_push/index.md:215]:
```suggestion
  process.env.VAPID_PRIVATE_KEY
```

File: files/es/web/progressive_web_apps/tutorials/js13kgames/re-engageable_notifications_push/index.md [hunk 14]
```diff
@@ -260,7 +260,7 @@ self.addEventListener("push", function (event) {
   event.waitUntil(
     self.registration.showNotification("ServiceWorker Cookbook", {
       body: payload,
-    })
+    }),
```
Comment (Graywolf9) [inline files/es/web/progressive_web_apps/tutorials/js13kgames/re-engageable_notifications_push/index.md:263]:
```suggestion
    })
```

File: files/es/web/xpath/introduction_to_using_xpath_in_javascript/index.md [hunk 15]
```diff
@@ -81,17 +91,37 @@ Obtenemos el valor regresado por la expresión accediendo a las siguientes propi
...
+  document,
+  null,
+  XPathResult.ANY_TYPE,
+  null,
+);
+
+alert(
+  "This document contains " +
+    paragraphCount.numberValue +
+    " paragraph elements",
```
Comment (Graywolf9) [inline files/es/web/xpath/introduction_to_using_xpath_in_javascript/index.md:105]:
```suggestion
    " paragraph elements"
```

File: files/es/web/xpath/introduction_to_using_xpath_in_javascript/index.md [hunk 16]
```diff
@@ -81,17 +91,37 @@ Obtenemos el valor regresado por la expresión accediendo a las siguientes propi
...
 ```js
-var paragraphCount = document.evaluate( 'count(//p)', document, null, XPathResult.ANY_TYPE, null );
-
-alert( 'This document contains ' + paragraphCount.numberValue + ' paragraph elements' );
+var paragraphCount = document.evaluate(
+  "count(//p)",
+  document,
+  null,
+  XPathResult.ANY_TYPE,
+  null,
```
Comment (Graywolf9) [inline files/es/web/xpath/introduction_to_using_xpath_in_javascript/index.md:99]:
```suggestion
  null
```

File: files/es/web/xpath/introduction_to_using_xpath_in_javascript/index.md [hunk 17]
```diff
@@ -40,7 +46,11 @@ Regresa `xpathResult`, que es un objeto `XPathResult` del tipo especificado en e
 Nosotros creamos un solucionador de namespace usando el método `createNSResolver` del objeto [document](/en/DOM/document).
 
 ```js
-var nsResolver = document.createNSResolver( contextNode.ownerDocument == null ? contextNode.documentElement : contextNode.ownerDocument.documentElement );
+var nsResolver = document.createNSResolver(
+  contextNode.ownerDocument == null
+    ? contextNode.documentElement
+    : contextNode.ownerDocument.documentElement,
```
Comment (Graywolf9) [inline files/es/web/xpath/introduction_to_using_xpath_in_javascript/index.md:52]:
```suggestion
    : contextNode.ownerDocument.documentElement
```

File: files/es/web/xpath/introduction_to_using_xpath_in_javascript/index.md [hunk 18]
```diff
@@ -13,7 +13,13 @@ La interfaz principal a usar con XPath es la función [evaluate](/en/DOM/documen
 Este método evalúa expresiones [XPath](/en/XPath) contra un documento basado en [XML](/en/XML) (incluyendo documentos HTML), y regresa un objeto [`XPathResult`](/en/XPathResult), el cual puede ser un solo nodo o un conjunto de nodos. La documentación existente para este método se encuentra en [document.evaluate](/en/DOM/document.evaluate), pero de momento es más bien escasa para nuestras necesidades; una examinación más abarcativa será dada a continuación.
 
 ```js
-var xpathResult = document.evaluate( xpathExpression, contextNode, namespaceResolver, resultType, result );
+var xpathResult = document.evaluate(
+  xpathExpression,
+  contextNode,
+  namespaceResolver,
+  resultType,
+  result,
```
Comment (Graywolf9) [inline files/es/web/xpath/introduction_to_using_xpath_in_javascript/index.md:21]:
```suggestion
  result
```

File: files/es/web/xpath/introduction_to_using_xpath_in_javascript/index.md [hunk 19]
```diff
@@ -81,17 +91,37 @@ Obtenemos el valor regresado por la expresión accediendo a las siguientes propi
...
 ```js
-var paragraphCount = document.evaluate('count(//p)', document, null, XPathResult.ANY_TYPE, null );
-
-alert( 'This document contains ' + paragraphCount.stringValue + ' paragraph elements' );
+var paragraphCount = document.evaluate(
+  "count(//p)",
+  document,
+  null,
+  XPathResult.ANY_TYPE,
+  null,
```
Comment (Graywolf9) [inline files/es/web/xpath/introduction_to_using_xpath_in_javascript/index.md:117]:
```suggestion
  null
```

File: files/es/web/xpath/introduction_to_using_xpath_in_javascript/index.md [hunk 20]
```diff
@@ -81,17 +91,37 @@ Obtenemos el valor regresado por la expresión accediendo a las siguientes propi
...
+);
+
+alert(
+  "This document contains " +
+    paragraphCount.stringValue +
+    " paragraph elements",
```
Comment (Graywolf9) [inline files/es/web/xpath/introduction_to_using_xpath_in_javascript/index.md:123]:
```suggestion
    " paragraph elements"
```

File: files/es/web/xpath/introduction_to_using_xpath_in_javascript/index.md [hunk 21]
```diff
@@ -120,18 +150,23 @@ Es importante señalar que si el documento se ha transformado (el arbol del docu
 ###### Iterator Example
 
 ```js
-var iterator = document.evaluate('//phoneNumber', documentNode, null, XPathResult.UNORDERED_NODE_ITERATOR_TYPE, null );
+var iterator = document.evaluate(
+  "//phoneNumber",
+  documentNode,
+  null,
+  XPathResult.UNORDERED_NODE_ITERATOR_TYPE,
+  null,
```
Comment (Graywolf9) [inline files/es/web/xpath/introduction_to_using_xpath_in_javascript/index.md:158]:
```suggestion
  null
```

File: files/es/web/xpath/introduction_to_using_xpath_in_javascript/index.md [hunk 22]
```diff
@@ -149,11 +184,16 @@ Snapshots do not change with document mutations, so unlike the iterators the sna
...
-
-for ( var i=0 ; i < nodesSnapshot.snapshotLength; i++ )
-{
-  dump( nodesSnapshot.snapshotItem(i).textContent );
+var nodesSnapshot = document.evaluate(
+  "//phoneNumber",
+  documentNode,
+  null,
+  XPathResult.ORDERED_NODE_SNAPSHOT_TYPE,
+  null,
```
Comment (Graywolf9) [inline files/es/web/xpath/introduction_to_using_xpath_in_javascript/index.md:192]:
```suggestion
  null
```

File: files/es/web/xpath/introduction_to_using_xpath_in_javascript/index.md [hunk 23]
```diff
@@ -171,9 +211,18 @@ Note that, for the unordered subtype the single node returned might not be the f
...
 ```js
-var firstPhoneNumber = document.evaluate('//phoneNumber', documentNode, null, XPathResult.FIRST_ORDERED_NODE_TYPE, null );
-
-dump( 'The first phone number found is ' + firstPhoneNumber.singleNodeValue.textContent );
+var firstPhoneNumber = document.evaluate(
+  "//phoneNumber",
+  documentNode,
+  null,
+  XPathResult.FIRST_ORDERED_NODE_TYPE,
+  null,
```
Comment (Graywolf9) [inline files/es/web/xpath/introduction_to_using_xpath_in_javascript/index.md:219]:
```suggestion
  null
```

File: files/es/web/xpath/introduction_to_using_xpath_in_javascript/index.md [hunk 24]
```diff
@@ -171,9 +211,18 @@ Note that, for the unordered subtype the single node returned might not be the f
...
+);
+
+dump(
+  "The first phone number found is " +
+    firstPhoneNumber.singleNodeValue.textContent,
```
Comment (Graywolf9) [inline files/es/web/xpath/introduction_to_using_xpath_in_javascript/index.md:224]:
```suggestion
    firstPhoneNumber.singleNodeValue.textContent
```

File: files/es/web/xpath/introduction_to_using_xpath_in_javascript/index.md [hunk 25]
```diff
@@ -193,7 +242,13 @@ The following code is intended to be placed in any JavaScript fragment within or
 To extract all the `<h2>` heading elements in an HTML document using XPath, the `xpathExpression` is simply '`//h2`'. Where, `//` is the Recursive Descent Operator that matches elements with the nodeName `h2` anywhere in the document tree. The full code for this is: link to introductory xpath doc
 
 ```js
-var headings = document.evaluate('//h2', document, null, XPathResult.ANY_TYPE, null );
+var headings = document.evaluate(
+  "//h2",
+  document,
+  null,
+  XPathResult.ANY_TYPE,
+  null,
```
Comment (Graywolf9) [inline files/es/web/xpath/introduction_to_using_xpath_in_javascript/index.md:250]:
```suggestion
  null
```

File: files/es/web/xpath/introduction_to_using_xpath_in_javascript/index.md [hunk 26]
```diff
@@ -247,17 +302,29 @@ req.send(null);
 
 var xmlDoc = req.responseXML;
 
-var nsResolver = xmlDoc.createNSResolver( xmlDoc.ownerDocument == null ? xmlDoc.documentElement : xmlDoc.ownerDocument.documentElement);
-
-var personIterator = xmlDoc.evaluate('//person', xmlDoc, nsResolver, XPathResult.ANY_TYPE, null );
+var nsResolver = xmlDoc.createNSResolver(
+  xmlDoc.ownerDocument == null
+    ? xmlDoc.documentElement
+    : xmlDoc.ownerDocument.documentElement,
```
Comment (Graywolf9) [inline files/es/web/xpath/introduction_to_using_xpath_in_javascript/index.md:308]:
```suggestion
    : xmlDoc.ownerDocument.documentElement
```

File: files/es/web/xpath/introduction_to_using_xpath_in_javascript/index.md [hunk 27]
```diff
@@ -247,17 +302,29 @@ req.send(null);
...
+);
+
+var personIterator = xmlDoc.evaluate(
+  "//person",
+  xmlDoc,
+  nsResolver,
+  XPathResult.ANY_TYPE,
+  null,
```
Comment (Graywolf9) [inline files/es/web/xpath/introduction_to_using_xpath_in_javascript/index.md:316]:
```suggestion
  null
```

File: files/es/web/xpath/introduction_to_using_xpath_in_javascript/index.md [hunk 28]
```diff
@@ -247,17 +302,29 @@ req.send(null);
...
 ```
 
 ### Note
 
 When the XPathResult object is not defined, the constants can be retreived in privileged code using `Components.interfaces.nsIDOMXPathResult.ANY_TYPE` (`CI.nsIDOMXPathResult`). Similarly, an XPathEvaluator can be created using:
 
 ```js
-Components.classes["@mozilla.org/dom/xpath-evaluator;1"].createInstance(Components.interfaces.nsIDOMXPathEvaluator)
+Components.classes["@mozilla.org/dom/xpath-evaluator;1"].createInstance(
+  Components.interfaces.nsIDOMXPathEvaluator,
```
Comment (Graywolf9) [inline files/es/web/xpath/introduction_to_using_xpath_in_javascript/index.md:326]:
```suggestion
  Components.interfaces.nsIDOMXPathEvaluator
```

File: files/es/web/xpath/introduction_to_using_xpath_in_javascript/index.md [hunk 29]
```diff
@@ -287,7 +354,13 @@ function nsResolver(prefix) {
 Our call to `document.evaluate` would then looks like:
 
 ```js
-document.evaluate( '//xhtml:td/mathml:math', document, nsResolver, XPathResult.ANY_TYPE, null );
+document.evaluate(
+  "//xhtml:td/mathml:math",
+  document,
+  nsResolver,
+  XPathResult.ANY_TYPE,
+  null,
```
Comment (Graywolf9) [inline files/es/web/xpath/introduction_to_using_xpath_in_javascript/index.md:362]:
```suggestion
  null
```

**Review by Graywolf9 — APPROVED** [review]
Thank you so much for this! I deleted some extra commas that the prettier put before some parenthesis

**Comment by queengooborg** [comment]
Hey @Graywolf9 -- trailing commas are the default in Prettier now, and upstream is using trailing commas in content, so we should be keeping them in for translated content as well!

**Comment by Graywolf9** [comment]
So there should be a comma before the closing parenthesis like

```
console.log("Some text",);
```

Sorry about this, I would revert it

**Comment by queengooborg** [comment]
> So there should be a comma before the closing parenthesis like
> 
> ```
> console.log("Some text",);
> ```
> 
> Sorry about this, I would revert it

Sort of -- if the arguments to a method are long enough to separate into multiple lines, then the last argument should have a comma after it.  Luckily, Prettier will automatically handle these cases and format the code accordingly, so we don't have to worry about the nuances!

**Comment by Graywolf9** [comment]
And when it is only one argument there should have the comma too? 

```
var nsResolver = document.createNSResolver(
  contextNode.ownerDocument == null
    ? contextNode.documentElement
    : contextNode.ownerDocument.documentElement
);
```

this method only has a promise as parameter
```
e.waitUntil(
    caches.open(cacheName).then(function (cache) {
      console.log(
        "[Servicio Worker] Almacena todo en caché: contenido e intérprete de la aplicación
      );
      return cache.addAll(contentToCache);
    })
);
```
```

**Comment by queengooborg** [comment]
It's more based on how many lines a function/array/object/etc. spans, rather than the number of arguments (see https://prettier.io/docs/en/options.html#trailing-commas).  The primary reason for trailing commas is to reduce merge conflicts if new arguments  are introduced, so that's why Prettier has made this change in default configuration!

**Comment by Graywolf9** [comment]
I see it, thank you so much for your time and explanation, and sorry about the missunderstood

**Review by Graywolf9 — APPROVED** [review]

**Comment by queengooborg** [comment]
Not a problem at all, I'm happy to help explain changes!  Thank you for reviewing and merging this PR!

**Pull request merged**